client/src/components/frontend/Project/Resources.js

```diff
@@ -3,12 +3,13 @@ import { ResourceList } from 'components/frontend';
 
 export default class ProjectResources extends Component {
 
-  static displayName = "Project.Resources"
+  static displayName = "Project.Resources";
 
   static propTypes = {
     project: PropTypes.object,
     resources: PropTypes.array,
-    pagination: PropTypes.object
+    pagination: PropTypes.object,
+    filterChange: PropTypes.func.isRequired
```

Comment:
Double check to be sure that everything currently using this component is passing this new required prop in.